src/exercises/day-3/safe-ffi-wrapper.rs

```diff
@@ -33,6 +36,18 @@ mod ffi {
         pub d_name: [c_char; 256],
     }
 
+    // Layout as per man entry for dirent
+    #[repr(C)]
+    #[cfg(target_os = "macos")]
```

Comment:
Great! Let's apply the attributes in the same ordre as above:

```suggestion
    #[cfg(target_os = "macos")]
    #[repr(C)]
```